crates/topos-tce-api/src/grpc/console.rs

```diff
@@ -1,60 +1,21 @@
-use std::{str::FromStr, sync::Arc};
+use std::sync::Arc;
+use tokio::sync::mpsc::Sender;
 
-use tokio::sync::{mpsc::Sender, oneshot, RwLock};
+use crate::runtime::InternalRuntimeCommand;
+use tokio::sync::RwLock;
 use tonic::{Request, Response, Status};
 use topos_core::api::grpc::tce::v1::{
-    console_service_server::ConsoleService, PushPeerListRequest, PushPeerListResponse,
-    StatusRequest, StatusResponse,
+    console_service_server::ConsoleService, StatusRequest, StatusResponse,
 };
-use topos_p2p::PeerId;
-
-use crate::runtime::InternalRuntimeCommand;
 
 pub(crate) struct TceConsoleService {
+    #[allow(dead_code)]
```

Comment:
Can we have a team rule that all uses `#[allow(dead_code)]` comes with a comment justifying it? Like `// Will be wired up as part of TP-xxx` or whatever happens to be the case.